cirq/pasqal/pasqal_qubits.py

```diff
@@ -192,3 +193,194 @@ def __rsub__(self, other: Tuple[int, int, int]) -> 'ThreeDGridQubit':
 
     def __neg__(self) -> 'ThreeDGridQubit':
         return ThreeDGridQubit(row=-self.row, col=-self.col, lay=-self.lay)
+
+
+class ThreeDQubit(cirq.ops.Qid):
+    """A qubit in 3d.
+
+    ThreeDQubits use z-y-x ordering:
+
+        ThreeDQubit(0, 0, 0) < ThreeDQubit(1, 0, 0)
+        < ThreeDQubit(0, 1, 0) < ThreeDQubit(1, 1, 0)
+        < ThreeDQubit(0, 0, 1) < ThreeDQubit(1, 0, 1)
+        < ThreeDQubit(0, 1, 1) < ThreeDQubit(1, 1, 1)
+
+    New ThreeDQubit can be constructed by adding or subtracting tuples
+
+        >>> cirq.pasqal.ThreeDQubit(2.5, 3, 4.7) + (3, 1.2, 6)
+        pasqal.ThreeDQubit(5.5, 4.2, 10.7)
+
+        >>> cirq.pasqal.ThreeDQubit(2.4, 3.1, 4) - (1, 2, 2.1)
+        pasqal.ThreeDQubit(1.4, 1.1, 1.9)
+    """
+
+    def __init__(self, x: float, y: float, z: float):
+        self.x = x
+        self.y = y
+        self.z = z
+
+    def _comparison_key(self):
+        return round(self.z, 9), round(self.y, 9), round(self.x, 9)
+
+    @property
+    def dimension(self) -> int:
+        return 2
+
+    def distance(self, other: cirq.ops.Qid) -> float:
+        """Returns the distance between two qubits in 3D."""
```

Comment:
Nit: In other places we use `3d`, `2d`. 
Having said that I realize that we have inconsistencies within Cirq - which I will probably cleanup :)